numpy/_core/tests/test_datetime.py

```diff
@@ -1390,6 +1390,16 @@ def test_timedelta_divmod(self, op1, op2):
         expected = (op1 // op2, op1 % op2)
         assert_equal(divmod(op1, op2), expected)
 
+    @pytest.mark.parametrize("op1, op2", [
+        # Y and M are incompatible with all units except Y and M
+        (np.timedelta64(1, 'Y'),
+         np.timedelta64(1, 's')),
+        (np.timedelta64(1, 'D'),
+         np.timedelta64(1, 'M')),    
```

Comment:
```suggestion
        (np.timedelta64(1, 'Y'), np.timedelta64(1, 's')),
        (np.timedelta64(1, 'D'), np.timedelta64(1, 'M')),    
```
Also just seems unnecessary (I don't think even black would add the newline).  But also just a silly style nit :).